website/versioned_docs/version-7.0.0/cli.md

```diff
@@ -100,15 +100,23 @@ npx babel src --out-file script-compiled.js
 
 ### Ignore files
 
-Ignore spec and test files
+Given files will be ignored from compilation and will not moved to destination folder.
 
 ```sh
 npx babel src --out-dir lib --ignore "src/**/*.spec.js","src/**/*.test.js"
 ```
 
+### Include ignored files
```

Comment:
I don't think we need this addition to the versioned doc since `--include-ignore` is new?